platform/platform-paxweb-jettyconfig/src/main/java/org/codice/ddf/pax/web/jetty/TraceContextFilter.java

```diff
@@ -15,24 +15,29 @@
 
 import java.io.IOException;
 import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import org.apache.shiro.util.ThreadContext;
+import org.codice.ddf.platform.filter.AuthenticationException;
+import org.codice.ddf.platform.filter.SecurityFilter;
+import org.codice.ddf.platform.filter.SecurityFilterChain;
 import org.codice.ddf.platform.filter.http.HttpFilter;
 import org.codice.ddf.platform.filter.http.HttpFilterChain;
-import org.codice.ddf.security.util.ThreadContextUtils;
+import org.codice.ddf.security.util.ThreadContextProperties;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /**
- * This servlet filter inserts a trace-id nto the shiro {@link ThreadContext} so it can be forwarded
- * to useful areas of interest.
+ * This servlet filter inserts a trace-id into the shiro {@link ThreadContext} so it can be
+ * forwarded to useful areas of interest.
  *
  * <p>This filter supports but doesn't full implement the W3C Trace Context specification. Refer to
  * <a href="https://www.w3.org/TR/trace-context/">W3C Trace Context Specification</a> and
  * specifically the <a href="https://www.w3.org/TR/trace-context/#trace-id">trace-id</a> property.
  */
-public class TraceContextFilter implements HttpFilter {
+public class TraceContextFilter implements HttpFilter, SecurityFilter {
```

Comment:
Made the same change to the trace context filter to ensure the trace ID is present during the security filter chain.